index.js

```diff
@@ -3,6 +3,14 @@
 // you.
 if (Number(process.version.slice(1).split(".")[0]) < 12) throw new Error("Node 12.0.0 or higher is required. Update Node on your system.");
 
+try {
+  require("./config.js");
+} catch {
+  console.log("Creating a config.js file for the bot");
+  require("./setup.js");
```

Comment:
After running the setup the app will have to restart, so within setup.js you can process.exit() in order to make sure the bot is restarted again.

Reply:
Resolved in 2716fe174311e9a54a1a8997ab1e247b94a8fa8a